prov/gni/src/gnix_av.c

```diff
@@ -400,7 +400,8 @@ static int map_reverse_lookup(struct gnix_fid_av *av_priv,
 
 			if ((index >= 0) && (index < entry->rx_ctx_cnt)) {
 				/* we have a match */
-				rx_addr = *(fi_addr_t *)&entry->gnix_addr;
+				memcpy(&rx_addr, &entry->gnix_addr,
```

Comment:
I don't think so.  Just using memcpy rather than a dereference.  Or maybe I"m confused.  This apparently is one of the recommended ways to suppress the alias warning the original code generated.